plugins/mcreator-localization/help/default/particle/max_age_diff.md

```diff
@@ -1 +1 @@
-This parameter controls how much actual maximal particle age can vary in positive or negative value from the value above.
\ No newline at end of file
+This parameter controls how much actual maximal particle age can vary in positive or negative value from the number above.
```

Comment:
Replace `the number above` with `specified max age` or similar so if GUI layout changes, the tip is still valid.

Reply:
:heavy_check_mark: Fixed